src/commcare_cloud/terraform/modules/iam/main.tf

```diff
@@ -24,3 +24,4 @@ resource "aws_iam_policy" "force_mfa" {
   description = "This policy allows users to manage their own passwords and MFA devices but nothing else unless they authenticate with MFA"
   policy = "${file("${path.module}/EnforceMFAPolicy.json")}"
 }
+
```

Comment:
nit: this looks like a stray blank line at the end of the file / unrelated change